tests/js/spec/views/settings/organizationDeveloperSettings/subscriptionBox.spec.jsx

```diff
@@ -34,4 +36,39 @@ describe('SubscriptionBox', () => {
     wrapper.setProps({disabled: true});
     expect(wrapper.find('Tooltip').prop('disabled')).toBe(false);
   });
+
+  describe('error.created resource subscription', () => {
+    beforeEach(() => {
+      onChange = jest.fn();
+      wrapper = mount(
+        <SubscriptionBox
+          resource="error"
+          checked={false}
+          disabled={false}
+          onChange={onChange}
+          organization={org}
+        />,
+        TestStubs.routerContext()
+      );
+    });
+
+    it('checkbox disabled without integrations-event-hooks flag', () => {
+      expect(wrapper.find('Tooltip').prop('disabled')).toBe(false);
+    });
+
+    it('checkbox visible with integrations-event-hooks flag', () => {
+      org = TestStubs.Organization({features: ['integrations-event-hooks']});
+      wrapper = mount(
+        <SubscriptionBox
+          resource="error"
+          checked={false}
+          disabled={false}
+          onChange={onChange}
+          organization={org}
+        />,
+        TestStubs.routerContext()
+      );
+      expect(wrapper.find('Tooltip').prop('disabled')).toBe(true);
```

Comment:
Shouldn't it be enabled? (opposed to this asserting that it's disabled)

Reply:
I've updated the tests to be clearer but when the `Checkbox` is enabled we want the `Tooltip` to be disabled and vice versa.